staff/enric-pedros/precourse/bingo.js

```diff
@@ -0,0 +1,117 @@
+
+
+
+function usuario() { //nombre de usuario
+    var usuarionuevo = prompt("Buenos dias, indique su nombre: ");
+    return usuarionuevo;
+}
+
+var cartro = []
+var lineasCartro = 5
+
+
+function aleatorio(a, b) {   //esta funcion genera un numero aleatorio entre un minimo y un máximo 
+    var num = Math.round(Math.random() * (b - a) + (a));
+    if (cartro.length === 0){
+        cartro.push({ number: num, matched: false })
+    }
+    else if (cartro.length < lineasCartro){
+        cartro.includes(num) ? '' : cartro.push({ number: num, matched: false });
+    }
+    if (cartro.length < lineasCartro){
+        
+        aleatorio(a,b)
+    }
+    
+} 
+
+
+
+var confirmacio = false
+var arrayNumeros

```

Comment:
the same as before, end the declarations with ";"

Reply:
do it!!!! ty